arches/app/templates/views/components/search/advanced-search.htm

```diff
@@ -116,29 +116,48 @@
-                        'aria-controls': uid,
-                        'aria-label': $root.translations.resourceTypeSearchFacets(name),
-                    }
+                click: function () { collapsed(!collapsed()) },
+                attr: {
+                    'aria-expanded': (!collapsed()).toString(),
+                    'aria-controls': uid,
+                    'aria-label': $root.translations.resourceTypeSearchFacets(name),
+                }
                 ">
-                    <div class="search-facet-item-heading">
-                        <strong data-bind="text: name"></strong>
-                        <span><i class="fa report-expander print-hide" data-bind="css: {'fa-angle-down': !collapsed(), 'fa-angle-right': collapsed()}"></i></span>
+                <div class="search-facet-item-heading">
+                    <strong data-bind="text: name"></strong>
+                    <span><i class="fa report-expander print-hide" data-bind="css: {'fa-angle-down': !collapsed(), 'fa-angle-right': collapsed()}"></i></span>
                     </div>
                 </div>
                 <div data-bind="visible: !collapsed()" role="list">
                     <!-- ko foreach: cards -->
-                    <div role="listitem">
-                        <a href="#" class="search-facet-item" role="button" data-bind="onEnterkeyClick, onSpaceClick, 
-                            click: addFacet, 
-                            attr: {
-                                id: uid,
-                                'aria-label': $root.translations.addCardToSearchQuery(name, $parent.name)
-                            }">
-                            <p class="search-facet-item-heading" data-bind="text: name"></p>
-                        </a>
+                    <div role="listitem" style="display: flex; align-items:center;">
+                        <div class="search-facet-item" style="display: flex;">
+                            <div>
+                                <div style="margin-right:5px;" role="button" data-bind="onEnterkeyClick, onSpaceClick, click: self.getNodegroupPath, class: $data.nodegroup_id == self.selectedNodegroupId() && self.expandPath() ? 'fa fa-times-circle' : 'fa fa-info-circle'"></div>
+                            </div>
+                            <div>
+                                <!-- ko if: $data.nodegroup_id == self.selectedNodegroupId() && self.expandPath() -->
+                                    <div>
+                                    <!-- ko if: self.cardPath().length > 0 -->
+                                        <!-- ko foreach: { data: self.cardPath(), as: 'val' } -->
+                                        <div style="display:flex;">
+                                            <div data-bind="style: { 'border-bottom': '10px', 'width': function() { return ($index() * 12) + 'px'}}"></div>
+                                            <div style="width: max-content;" data-bind="text: val"></div>
```

Comment:
The font size of the path is actually a little bit bigger than the name of the facet. I think it would be better if the path were the same font size or smaller. It might also be nice to make it a little lighter in color. Something like this might look good: 
```
    color: #777;
    font-size: 1.3rem;
```